db/transactions.js

```diff
@@ -1,20 +1,23 @@
 import conn from './connection'
+import { getUser } from './users'
+import { getProduct } from './products'
 
 // USERS
 conn.exec(`CREATE TABLE IF NOT EXISTS transactions (
     id INTEGER PRIMARY KEY AUTOINCREMENT,
     user_id VARCHAR,
     amount_cents INTEGER NOT NULL DEFAULT 100,
     product_id INTEGER DEFAULT NULL,
+    product_name VARCHAR DEFAULT NULL,
```

Comment:
Remove `product_id` and rename `product_name` to `name`. This keeps it more generic.